### PR TITLE
README - English Edits

### PR DESCRIPTION
**Pull request by ahandsel**
## Why

* Improve the README file's English
* Clarify the installation process

## What

Changes made to README:
* Capitalize Kintone's K
* Add shell or text to unspecified code blocks
* Installation Section > Add move to `/usr/local/bin` step
* Installation > Append a PATH environment variables note
* Replace curly brackets
* English review
* Table Table of Contents {New}
* Updated Installation's Step 3, 4, & Notes {New}

## How to test

* Confirm the changes on the README file locally

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/cli-kintone/blob/main/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `yarn lint` and `yarn test` on the root directory.

### Warning from Linters
The following got flagged by the linter but were not the result of this PR:

[lint:eslint] ✖ 5 problems (0 errors, 5 warnings)

`warning ../../package.json: No license field`

`cli-kintone/src/__tests__/api.test.ts`
* 226:19  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap
* 251:19  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap

`cli-kintone/src/kintone/client.ts`
* 65:10  warning  A function with a name starting with an uppercase letter should only be used as a constructor  new-cap

`cli-kintone/src/record/export/usecases/__tests__/get/index.test.ts`
* 92:28  warning  Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion

`cli-kintone/src/record/import/usecases/add/record.ts`
* 84:13  warning  Blocks are nested too deeply (5). Maximum allowed is 4  max-depth

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,18 @@ A CLI tool to import/export records to/from [kintone](https://www.kintone.com/).
    - Windows: `cli-kintone-win.zip`
    - Linux: `cli-kintone-linux.zip`
    - macOS: `cli-kintone-macos.zip`
-3. Extract executables from the ZIP file and run it.
+3. Extract the downloaded zip file and move the `cli-kintone` file to the `/usr/local/bin`.
+4. Run the `cli-kintone` command and confirm that the command is available.
+
+Note: If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
```
Comment (mshrtsr) [inline README.md:29]: Currently, we don't instruct to place cli-kintone file to directory registered in `$PATH`.
There are several reasons.

- It is an optional step to be able to execute cli-kintone from anywhere and not a required step
- The `$PATH` depends on the users' OS and settings.
- cli-kintone is mainly used on Windows. There is no default directory like `/usr/local/bin` on Windows.

Thus, we want to keep it as is without adding to `$PATH` step.
However, I thought it is nice to separate the extraction process and execution process as you edited.

```suggestion
3. Extract the downloaded zip file
4. Run the extracted `cli-kintone.exe` on command prompt (Windows), or run `cli-kintone` on your terminal (Linux/macOS).
```

**Review by tasshi-me — CHANGES_REQUESTED** [review]
Thank you for contributing a Pull Request!

I left some comments to request changes.
Could you check them, please?

File: README.md (hunk 1 above)
Comment (jesslynvc) [reply README.md:29]: I agree with removing the /usr/local/bin directory, but I think we should leave the note.
```
Note: If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
```

In my case, when executing [cli-kintone v1.3.2](https://github.com/kintone/cli-kintone/releases/tag/v1.3.2), I could not run `cli-kintone` on my terminal even though I was in the cli-kintone folder. I had to specify full path (ex: /Downloads/cli-kintone-for-mac/cli-kintone --version) or edit the environment path to be able to run it.

File: README.md [hunk 2]
```diff
@@ -154,7 +157,7 @@ Options:
 
 ##### Notes
 
-- The field in a Table cannot be specified to the `fields` option.
+- A field in a Table cannot be specified with the `fields` option.
```
[the remark this answers is not in the document]
Comment (jesslynvc) [reply README.md:160]: OKです！

File: README.md (hunk 1 above)
Comment (ahandsel) [reply README.md:29]: Okay, I understand that we need to remove the `and move the cli-kintone file to the /usr/local/bin` bit from Step 3.

Then I would like to keep the Note and add to it. Something like:

```md
Note:
* If the terminal responds with an error stating that the command could not be found, the PATH environment variables may need to be edited to use cli-kintone.
* For Linux and macOS users, it is recommended to move the `cli-kintone` file to the `/usr/local/bin`.
```

Comment (mshrtsr) [reply README.md]: We decided:

- Add `./` on the fourth step of the installation
- Add a note to run cli-kintone on any directory

**Review by tasshi-me — APPROVED** [review]
LGTM 👍

**Comment by mshrtsr** [comment]
TODO: merge with `docs` prefix.

**Pull request merged**